docs/windows.md

```diff
@@ -0,0 +1,64 @@
+# Working on Windows
+
+Jujutsu works the same on all platforms, but there are some caveats that Windows
+users should be aware of.
+
+## Line endings are not converted
+
+Jujutsu does not honor `.gitattributes` and does not have a setting like Git's
+`core.autocrlf`. This means that line endings will be checked out exactly as
+they are committed and committed exactly as authored. This is true on all
+platforms, but Windows users are most likely to miss CRLF conversion.
+
+## Pagination
+
+[Pagination is disabled by default on Windows][issue-2040] because Windows
+doesn't ship with a usable pager.
+
+If you have Git installed, you can use Git's pager and re-enable pagination:
+
+```powershell
+PS> jj config set --user ui.pager '["C:\\Program Files\\Git\\usr\\bin\\less.exe", "-FRX"]'
+PS> jj config set --user ui.paginate auto
+```
+
+## Typing `@` in PowerShell
+
+PowerShell uses `@` as part the [array sub-expression operator][array-op], so it
+often needs to be escaped or quoted in commands:
+
+```powershell
+PS> jj log -r `@
```

Comment:
Just a question, this singular \` works for Windows? At work I've been mostly using `jj log -r "@"`.

Reply:
Yes, backtick is the escape character in PowerShell (newline is ``` `n ```, etc.):

```
> jj log -r `@
@  qyqrpwvz stephen.g.jennings@gmail.com 16 minutes ago windows-docs default@ 1a3b8731
│  docs: add notes for Windows users
~
```

Reply:
TIL, thanks for clarifying.